docs/faq/proof-of-stake/stake-pools.md

```diff
@@ -4,7 +4,7 @@
 
 #### **1. Are there any other benefits to a stake pool other than not needing to run a full node and keeping your wallet unlocked? For example, will it have a better chance of winning or anything like that?**
 
-Well first keep in mind that wallet (`dcrwallet`) != core network (`dcrd`). In order to stake you need to have `dcrwallet` online and unlocked 24/7.
+Well first keep in mind that the wallet (`dcrwallet`) is not the core network (`dcrd`). In order to stake you need to have `dcrwallet` online and unlocked 24/7.
 
 With that out of the way, you are correct that the only real benefit of the current pool is that it will provide worry-free voting since it provides multi-location redundancy and automatic failure recovery[^9262], neither of which your local setup is very likely to have.
 
```

Comment:
It can be true that a well configured pool will have a lower miss rate on account of lower latency from the voting wallet to the miners.  This hasn't been a problem for months, but some miners/pools were very... "impatient" when waiting for votes to come in.  The result was that there were misses when votes were actually transmitted, just with not the lowest latency of the bunch.